.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,8 @@ _Have you read the [devfile registry contributing guide](https://github.com/devf
 _Does this repository's tests pass with your changes?_
 - [ ] Documentation
 _Does any documentation need to be updated with your changes?_
+- [ ] Check Tools Provider
+_Does this change impact existing tools, i.e. Odo, Che, Console? If it does, have you tested the change with those tools?_
```

Comment:
We may want to provide a link to provide more info on that so that the developer knows how to answer this question properly.

Reply:
talked with Tomas and Mario, they both suggest any change to registry need to be verified with tools to ensure it won't break tools. Reword the sentence to clarify the requirement to test with tools. Also updated the contributing.md to include instruction on how to test with existing tools providers.